src/content/self-hosted/administration/configuration.mdx

```diff
@@ -623,6 +483,137 @@ registry:
         size: 40Gi
 ```
 
+##### azure
+
+Use this to use Azure Storage to store your private images.
+
+```yaml
+registry:
```

Comment:
why do we have a double `storage` key?

Reply:
One is the fixed `storage` section of the registry, the other one is for `azure` storage service.
For simplicity, we are keeping the same format between `cloud.provider` and `registry.storage.provider`